src/aics-image-viewer/shared/utils/playControls.ts

```diff
@@ -0,0 +1,80 @@
+import { AxisName } from "../types";
+
+type PlayAxisName = AxisName | "t";
+
+const PLAY_STEP_INTERVAL = 125;
```

Comment:
Nit: I think it would be worth adding units (milliseconds) to this constant name

```suggestion
const PLAY_STEP_INTERVAL_MS = 125;
```